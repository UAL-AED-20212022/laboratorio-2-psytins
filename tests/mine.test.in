RPI Portugal
RPI Espanha
RPF Italia
RPF Angola
RPI Argentina
RPDE França Espanha
RPAE Moçambique Portugal
RPII Índia 3
VNE
RPI Canadá
EPE
RPI Holanda
EUE
EP Índia
EP Loisina
EUE
VNE
VP Espanha
VP Portugal
VP Jolanda