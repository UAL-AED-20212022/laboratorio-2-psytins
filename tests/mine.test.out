Portugal  
None
Espanha  
Portugal  
None
Espanha  
Portugal  
Italia  
None
Espanha  
Portugal  
Italia  
Angola  
None
Argentina  
Espanha  
Portugal  
Italia  
Angola  
None
<models.Node.Node object at 0x7faafba53130>
Argentina  
Espanha  
França  
Portugal  
Italia  
Angola  
None
<models.Node.Node object at 0x7faafba53130>
Argentina  
Espanha  
França  
Moçambique  
Portugal  
Italia  
Angola  
None
Argentina  
Espanha  
Índia  
França  
Moçambique  
Portugal  
Italia  
Angola  
None
O número de elementos são 8.
Canadá  
Argentina  
Espanha  
Índia  
França  
Moçambique  
Portugal  
Italia  
Angola  
None
O país Canadá foi eliminado da lista.
Holanda  
Argentina  
Espanha  
Índia  
França  
Moçambique  
Portugal  
Italia  
Angola  
None
O país Angola foi eliminado da lista.
Item found
O país Angola foi eliminado da lista.
item bot found
O país Loisina não se encontra na lista.
O país Italia foi eliminado da lista.
O número de elementos são 6.
Item found
O país Espanha encontra-se na lista.
Item found
O país Portugal encontra-se na lista.
item bot found
O país Jolanda não se encontra na lista.
